src/test/regress/expected/local_shard_utility_command_execution.out

```diff
@@ -0,0 +1,729 @@
+-- This tests file includes tests for local execution of utility commands.
+-- For now, this file includes tests only for local execution of
+-- `TRUNCATE/DROP/DDL` commands for all kinds of distributed tables from
+-- the coordinator node having regular distributed tables' shards
+-- (shouldHaveShards = on) and having reference table placements in it.
+\set VERBOSITY terse
+SET citus.next_shard_id TO 1500000;
+SET citus.shard_replication_factor TO 1;
+SET citus.enable_local_execution TO ON;
+SET citus.shard_COUNT TO 32;
+SET citus.log_local_commands TO ON;
+CREATE SCHEMA local_commands_test_schema;
+SET search_path TO local_commands_test_schema;
+-- let coordinator have distributed table shards/placements
+SELECT master_add_node('localhost', :master_port, groupId => 0);
+ master_add_node
+---------------------------------------------------------------------
+         1380010
+(1 row)
+
+SELECT master_set_node_property('localhost', :master_port, 'shouldhaveshards', true);
+ master_set_node_property
+---------------------------------------------------------------------
+
+(1 row)
+
+---------------------------------------------------------------------
+------ local execution of TRUNCATE ------
+---------------------------------------------------------------------
+CREATE TABLE ref_table (a int primary key);
+SELECT create_reference_table('ref_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE dist_table(a int);
+SELECT create_distributed_table('dist_table', 'a');
```

Comment:
Minor concern: Since `create_distributed_table ` favors colocation, if it finds any colocated tables in the cluster, it'd not use coordinator for the shards. 

Shall  we do `SELECT create_distributed_table('dist_table', 'a', colocate_with:='none');?`

But, please make sure that all the other distributed tables created in this file are colocated with this distributed table. If we drop this table, use `colocate_with:='none'` for the other distributed tables, else use colocate_with:='dist_table'

In fact, this is the primary reason that this test fails with `check-base`.

So, could you aim making the tests pass with `check-base`?

Reply:
makes sense, I even didn't know this fact, thanks

Reply:
fixed, now `make check-base EXTRA_TESTS=local_shard_utility_command_execution` fails only with "replicated reference tables", which are left over from the tests within the schedule, naturally